lib/recording/RecordingManager.js

```diff
@@ -380,6 +397,8 @@ class RecordingManager {
                     return sessionObj.session.verifyFlavorSetMatchPlaylistFlavors(entryObj)
                         .then((flavorsMatch) => {
                             if (!flavorsMatch) {
+                                logger.info(`[handleFlavorsMismatch] set lock mp4FilesQueueLock`);
+                                sessionObj.session.mp4FilesQueueLock = true;
```

Comment:
why we need the lock? the flavordownloader hasn't started yet

Reply:
yes. not needed. removed